rules-java-archives/addon/src/main/java/org/jboss/windup/rules/apps/java/archives/listener/ArchiveIdentificationGraphChangedListener.java

```diff
@@ -20,24 +19,34 @@
 import com.tinkerpop.blueprints.Edge;
 import com.tinkerpop.blueprints.Vertex;
 import com.tinkerpop.blueprints.util.wrappers.event.listener.GraphChangedListener;
+import javax.inject.Inject;
+import org.jboss.windup.rules.apps.java.archives.identify.SortedFilesIdentifier;
 
 /**
  * {@link GraphChangedListener} responsible for identifying {@link ArchiveModel} instances when they are added to the
  * graph.
- * 
+ *
  * @author <a href="mailto:lincolnbaxter@gmail.com">Lincoln Baxter, III</a>
  */
 public class ArchiveIdentificationGraphChangedListener implements GraphChangedListener
 {
-    private static final Logger log = Logger.getLogger(ArchiveIdentificationGraphChangedListener.class.getSimpleName());
+    private static final Logger log = Logger.getLogger(ArchiveIdentificationGraphChangedListener.class.getName());
+
+
+    @Inject private SortedFilesIdentifier identifier;
+
 
     private GraphRewrite event;
 
     public ArchiveIdentificationGraphChangedListener(GraphRewrite event)
     {
         this.event = event;
+
+        //FurnaceHolder.getFurnace().getAddonRegistry().getServices(SortedFilesIdentifier.class);
```

Comment:
I wanted to ask how to get a particular instance of the class, and use CDI instead of static  getInstance().